docs/pages/config/simple_example.mdx

```diff
@@ -48,45 +83,45 @@ This will generate 2 server functions, the default one and the edge one. The edg
 Edge runtime function have less cold start time, but you can only deploy one route per function. They also does not have the middleware bundled in the function, so you need to use external middleware if you need it in front of the edge function.
```

Comment:
While you're here, can you also update this:

```suggestion
Edge runtime function have less cold start time, but you can only deploy one route per function. They also do not have the middleware bundled in the function, so you need to use external middleware if you need it in front of the edge function.
```

Reply:
for sure. adding another commit